controller-server/src/main/java/com/yahoo/vespa/hosted/controller/persistence/VersionStatusSerializer.java

```diff
@@ -105,6 +107,7 @@ private VespaVersion vespaVersionFromSlime(Inspector object) {
                                 Instant.ofEpochMilli(object.field(committedAtField).asLong()),
                                 object.field(isControllerVersionField).asBool(),
                                 object.field(isSystemVersionField).asBool(),
+                                object.field(isReleasedField).asBool(),
```

Comment:
You're ok with the default here if missing?

Reply:
While the controllers upgrade to include this for the first time, the upgraded ones may fail to provide a compile version some of the time. I think this is acceptable.